### ISSUE TITLE
img tags with alt attribute with spaces are getting translated incorrectly when using GitHub dialect

### ISSUE THREAD

**Issue by shlomif** [houseabsolute/Markdent#21]
Hi!

I have some `<img` tags with `align="right"` (or "left") and alt attributes with spaces in the README.md here (mostly SFW but with some photos of celebs):

https://github.com/shlomif/my-real-person-fan-fiction

if I convert the `&nbsp;` to spaces, the img is translated badly:

```

shlomif[homepage]:$trunk$ markdent-html --dialects GitHub --title 'foo' --file lib/repos/my-real-person-fan-fiction/README.md
<!DOCTYPE html>
<html><head><title>foo</title></head><body><h1>Why I will continue to write my real person fanfiction (fanfic)
</h1><h2>TL;DR Summary
</h2><p>I, <a href="https://www.shlomifish.org/">ShlomiFish</a>, writer, software developer, and
amateur philosopher, will continue to write real person fanfiction, even though
some people told me I must not.  However, I'd like to note why it may have been
a blessing in disguise.
</p><h2>The Technion vs. Project Euler
</h2><p><img Euler" Leonhard align="right" alt="&quot;Portrait" of src="./euler.webp">
</p><p>I joked that my opinion on <a href="https://en.wikipedia.org/wiki/Technion_%E2%80%93_Israel_Institute_of_Technology">The
Technion</a>,
the University where I completed my B.Sc degree in Electrical Engineering (=
&quot;EE&quot;) ,has become &quot;untwitterable&quot;. The more I think about it, the more it seems
that I think less highly of it, despite the fact that some people told me that
it wasn't a waste of time, and that I'll reminiscient about my studies. Part of
the problem was that I studied there only in order to get my B. Sc. in
Electrical Engineering (while I'm not interested in electronics), which I
wanted in order to get more higher-paying jobs.
shlomif[homepage]:$trunk$ 

```

given from this markdown source code:

```
shlomif[homepage]:$trunk$ cat lib/repos/my-real-person-fan-fiction/README.md
# Why I will continue to write my real person fanfiction (fanfic)

## TL;DR Summary

I, [Shlomi Fish](https://www.shlomifish.org/), writer, software developer, and
amateur philosopher, will continue to write real person fanfiction, even though
some people told me I must not.  However, I'd like to note why it may have been
a blessing in disguise.

## The Technion vs. Project Euler

<img align="right" alt="Portrait of Leonhard Euler" src="./euler.webp" />

I joked that my opinion on [The
Technion](https://en.wikipedia.org/wiki/Technion_%E2%80%93_Israel_Institute_of_Technology),
the University where I completed my B.Sc degree in Electrical Engineering (=
"EE") ,has become "untwitterable". The more I think about it, the more it seems
that I think less highly of it, despite the fact that some people told me that
it wasn't a waste of time, and that I'll reminiscient about my studies. Part of
the problem was that I studied there only in order to get my B. Sc. in
Electrical Engineering (while I'm not interested in electronics), which I
wanted in order to get more higher-paying jobs.
shlomif[homepage]:$trunk$ 
```

Thanks!

**Comment by autarch** [comment]
Hi, Shlomi,

The content you pasted was treated as markdown and formatted by GitHub, so I can't see it.

But it'd be much easier to fix this if you can give me a simple test case.

**Comment by shlomif** [comment]
> Hi, Shlomi,
> 
> The content you pasted was treated as markdown and formatted by GitHub, so I can't see it.
> 
> But it'd be much easier to fix this if you can give me a simple test case.

I corrected it, but per you request:

```
markdent-html --dialects GitHub --title 'foo' --text=$'# Test\n\n<img src="dummy.png" alt="text with spaces" align="right" />\n'
```

gives me:

```
<!DOCTYPE html>
<html><head><title>foo</title></head><body><h1>Test
</h1><p><img align="right" alt="&quot;text" spaces" src="dummy.png" with>
</p></body></html>

```

**Comment by autarch** [comment]
Thanks for the report! Fixed in v0.35.

**Comment by shlomif** [comment]
> Thanks for the report! Fixed in v0.35.

Thanks! I will try it and let you know..

**Comment by shlomif** [comment]
Works nicely, thanks!
